source/tests/tf/test_data_modifier.py

```diff
@@ -57,7 +57,9 @@ def _setUp(self):
             restart=None, init_model=None, log_path=None, log_level=30, mpi_log="master"
         )
         jdata = j_loader(INPUT)
-
+        jdata["model"]["fitting_net"]["embedding_width"] = jdata["model"]["descriptor"][
+            "neuron"
+        ][-1]
```

Comment:
This should not be required. `DPTrainer` should be able to add it automatically. If not, there is a bug.

Reply:
right.